e2e/container/run_test.go

```diff
@@ -68,7 +68,6 @@ func TestRunAttach(t *testing.T) {
 			}
 
 			assert.Equal(t, c.ProcessState.ExitCode(), 7)
-			assert.Check(t, is.Contains(d.String(), "exit status 7"))
```

Comment:
I guess if we want to be strict in our tests, we could consider checking for the error message to be empty (with a comment);


```go
assert.Check(t, is.Equal(err.Error(), ""), "should not print our own error message, as the container's output is the error message")
```

Not urgent for how, but something we could consider

Reply:
Derp; meant checking the `d.String()` for what we expect it to output (for that we could add a `echo "something failed"` to the container's command.


(but again, probably fine to look at in a follow up)